### DISCUSSION TITLE
Client side approval before Unattended connection

### DISCUSSION THREAD

**Discussion by lazee486** [rustdesk/rustdesk#10073] [Feature Request]
when unattended is setup especially in commercial, clients get upset when you instantly connect, even when they asked you to.
common issue is maybe having banking etc open. 

solution: a 10 second popup warning them of incoming connection with an option to refuse (possibly few preset reasons) 

if no deny happens it connects anyway (so you can use for genuine unattended use cases)  

following the same keep client informed, after disconnecting a popup remains saying who connected for how long.

**Comment by rustdesk** [comment]
We can consider add this, but we need to listen to more voices first.

> solution: a 10 second popup warning them of incoming connection with an option to refuse (possibly few preset reasons)
